lib/listcoef.gi

```diff
@@ -262,73 +262,50 @@ end );
 
 #############################################################################
 ##
-#M  MultRowVector( <list1>, <poss1>, <list2>, <poss2>, <mult> )
+#M  MultVectorLeft( <list>, <mul> )
 ##
-InstallMethod( MultRowVector,"generic method",
+InstallMethod( MultVectorLeft,
+    "for a mutable dense list, and a multiplicative element",
     true,
     [ IsDenseList and IsMutable,
-      IsDenseList,
-      IsDenseList,
-      IsDenseList,
       IsMultiplicativeElement ],
     0,
```

Comment:
While you are at it, you might just as well remove the `true` and `0` here, as they are superfluous.

Reply:
Same for the other `InstallMethod` calls you change